terraform/openstack/hosts/main.tf

```diff
@@ -64,3 +64,166 @@ resource "openstack_compute_instance_v2" "resource" {
   }
   count = "${ var.resource_count }"
 }
+
+resource "openstack_compute_secgroup_v2" "secgroup_consul_common" {
+  name = "${var.short_name}-consul-common"
+  description = "security group for common ports between agents and servers"
+
+  rule { # HTTP-API
+    from_port = 8500
+    to_port = 8500
+    ip_protocol = "tcp"
+    cidr = "0.0.0.0/0"
```

Comment:
Can we use `self`for the traffic between the nodes inside the cluster instead of a wide open `0.0.0.0/0`? See the docs at 
https://terraform.io/docs/providers/openstack/r/compute_secgroup_v2.html